src/util.js

```diff
@@ -27,16 +27,17 @@ export function getWindowHeight() {
 // Get the highest window context that isn't cross-origin
 // (When in an iframe)
 export function getHighestSafeWindowContext(self = global.window.self) {
+  const referrer = self.document.referrer;
```

Comment:
Maybe it is better to use destructuring assignment?
```javascript
const { referrer } = self.document;
```

Reply:
You're right, just pushed!